lxd/instance/drivers/driver_lxc.go

```diff
@@ -8371,6 +8371,16 @@ func (d *lxc) Metrics(hostInterfaces []net.Interface) (*metrics.MetricSet, error
 	return out, nil
 }
 
+// EFIVarsRead does not make any sense for containers.
+func (d *lxc) EFIVarsRead() (api.InstanceEFI, error) {
```

Comment:
We dont need these as we wont be adding them to the general Instance interface.

Reply:
just a nit but looking at the other functions on the Instance interface and we've not tended to use `Get` or `Read` suffixes for the "getter" functions, instead just used the entity type, e.g. `LocalConfig()` or `Snapshots()` etc. So how about just `EFIVars()`?